src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1854,6 +1854,45 @@ limitations under the License.
             <include cluster="Relative Humidity Measurement" client="false" server="false" clientLocked="false" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-fan</name>
+        <domain>HA</domain>
+        <typeName>Matter Fan</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x003B</deviceId>
```

Comment:
I see the value as 0x002B in the spec. Can you please confirm?

Reply:
Oops!! Updated to 0x002B. Thank you